docs/resources/rds_pg_database.md

```diff
@@ -0,0 +1,105 @@
+---
+subcategory: "Relational Database Service (RDS)"
+---
+
+# huaweicloud_rds_pg_database
+
+Manages RDS PostgreSQL database resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+
+resource "huaweicloud_rds_pg_database" "test" {
+  instance_id = var.instance_id
+  name        = "test"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of the RDS PostgreSQL instance.
+
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the database name. The value contains 1 to 63 characters, including
+  letters, digits, and underscores (_). It cannot start with pg or a digit, and must be different from RDS for
+  PostgreSQL template library names. RDS for PostgreSQL template libraries include **postgres**, **template0**, and
+  **template1**.
+
+  Changing this parameter will create a new resource.
+
+* `owner` - (Optional, String, ForceNew) Specifies the database user. The value must be an existing username and must be
+  different from system usernames. Defaults to **root**.
+
+  Changing this parameter will create a new resource.
+
+* `template` - (Optional, String, ForceNew) Specifies the name of the database template. Value options: **template0**,
+  **template1**. Defaults to **template1**.
+
+  Changing this parameter will create a new resource.
+
+* `character_set` - (Optional, String, ForceNew) Specifies the database character set.
+  For details, see [documentation](https://www.postgresql.org/docs/16/infoschema-character-sets.html).
+  Defaults to **UTF8**.
+
+  Changing this parameter will create a new resource.
+
+* `lc_collate` - (Optional, String, ForceNew) Specifies the database collocation.
+  For details, see [documentation](https://support.huaweicloud.com/intl/en-us/bestpractice-rds/rds_pg_0017.html).
+  Defaults to **en_US.UTF-8**.
+
+-> **NOTE:** For different collation rules, the execution result of a statement may be different.
```

Comment:
Please leave two spaces, if this document is a description of `lc_collate`, and the English description has grammatical errors.

Reply:
done